csharp/extractor/Semmle.Extraction.CSharp/SymbolExtensions.cs

```diff
@@ -434,7 +434,7 @@ public static bool IsSourceDeclaration(this IParameterSymbol parameter)
             return true;
         }
 
-        public static IEntity CreateEntity(this Context cx, ISymbol symbol)
+        public static IEntity? CreateEntity(this Context cx, ISymbol? symbol)
```

Comment:
Can we come up with a more null friendly error handling in this method? Ideally we should only return null if the input was null. We can't we return an `IEntity` that belongs to nulltype (`NullType.Create(cx).Type`) or the error type symbol?